src/Aquarius.ONE.ClientSDK/Enterprise/Twin/DigitalTwinApi.cs

```diff
@@ -1,645 +1,681 @@
+            digitalTwin.Name = name;
+            digitalTwin.CategoryId = 2;
+            digitalTwin.TwinReferenceId = Guid.NewGuid().ToString();
+            digitalTwin.TwinTypeId = twinTypeId;
+            digitalTwin.TwinSubTypeId = twinSubTypeId;
+            return await CreateAsync(digitalTwin);
+        }
+
+        public async Task<DigitalTwin> CreateAsync(DigitalTwin digitalTwin)
+        {
+            var watch = System.Diagnostics.Stopwatch.StartNew();
+            var requestId = Guid.NewGuid();
+            var endpoint = $"{EndpointRoot}/DigitalTwin";
+
+            JsonSerializerSettings jsonSettings = new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore
+            };
+            var json = JsonConvert.SerializeObject(digitalTwin, jsonSettings);
+
+            try
+            {
+                var respContent = await _restHelper.PostRestJSONAsync(requestId, json, endpoint).ConfigureAwait(_continueOnCapturedContext);
+                if (respContent.ResponseMessage.IsSuccessStatusCode)
+                {
+                    ApiResponse apiResponse = JsonConvert.DeserializeObject<ApiResponse>(respContent.Result, new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore });
+                    var results = apiResponse.Content.DigitalTwins.Items.Select(x => x).ToList();
+                    foreach (var result in results)
+                    {
+                        Event(null, new ClientApiLoggerEventArgs { EventLevel = EnumOneLogLevel.OneLogLevelTrace, HttpStatusCode = respContent.ResponseMessage.StatusCode, ElapsedMs = watch.ElapsedMilliseconds, Module = "DigitalTwinApi", Message = "CreateAsync Success" });
+                        return result;
+                    }
+                }
+                Event(null, new ClientApiLoggerEventArgs { EventLevel = EnumOneLogLevel.OneLogLevelWarn, HttpStatusCode = respContent.ResponseMessage.StatusCode, ElapsedMs = watch.ElapsedMilliseconds, Module = "DigitalTwinApi", Message = "CreateAsync Failed" });
+                return null;
+            }
+            catch (Exception e)
+            {
+                Event(e, new ClientApiLoggerEventArgs { EventLevel = EnumOneLogLevel.OneLogLevelError, Module = ModuleName, Message = $"CreateAsync Failed - {e.Message}" });
+                if (_throwAPIErrors)
+                     throw;
+                return null;
+            }
+        }
+
+        public async Task<List<DigitalTwin>> CreateManyAsync(List<DigitalTwin> digitalTwins)
+        {
+            var watch = System.Diagnostics.Stopwatch.StartNew();
+            var requestId = Guid.NewGuid();
+            var endpoint = $"{EndpointRoot}/DigitalTwin/many";
+
+            var twins = new DigitalTwins();
+            twins.Items.AddRange(digitalTwins);
+            var jsonSettings = new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore
+            };
```

Comment:
Nice to have tidy up, since the exact same serialization setting is used many times around this API class, it'd be nice to lift it as a common setting in the class.

Reply:
I feel like we have this available in one of our shared libraries somewhere.

Reply:
There is so much cleanup I'd like to do for these classes, but I had to pick my battles though and stop before rewriting the whole thing.

Reply:
I feel the same way every time I have to work in this repo.  Would love to have a week and carte blanche to go crazy on this one or the utilities.data repo